models/board/src/index.ts

```diff
@@ -70,6 +70,12 @@ export class TCardLabel extends TAttachedDoc implements CardLabel {
   isHidden?: boolean
 }
 
+@Model(board.class.CardChecklist, core.class.Obj, DOMAIN_MODEL)
+@UX(board.string.Checklist)
+export class TCardChecklist extends TObj implements CardChecklist {
```

Comment:
Why you need it in model?

Reply:
We would need to add presenters for this in the future to display it in activity